java/client/src/main/java/glide/api/BaseClient.java

```diff
@@ -100,4 +96,82 @@ protected static ConnectionManager buildConnectionManager(ChannelHandler channel
     protected static CommandManager buildCommandManager(ChannelHandler channelHandler) {
         return new CommandManager(channelHandler);
     }
+
+    /**
+     * Extracts the response from the Protobuf response and either throws an exception or returns the
+     * appropriate response as an <code>Object</code>.
+     *
+     * @param response Redis protobuf message
+     * @return Response <code>Object</code>
+     */
+    protected Object handleObjectResponse(Response response) {
```

Comment:
Do you want to move all handler updates into #915?

Reply:
I'll add them as needed, and fix conflicts as they arise.  That would be better.